src/Sylius/Bundle/AdminBundle/templates/customer/order/index/content/header/title_block/title.html.twig

```diff
@@ -0,0 +1,10 @@
+{% import '@SyliusAdmin/shared/helper/header.html.twig' as header %}
+
+{% set configuration = hookable_metadata.context.resources.requestConfiguration %}
+{% set customer = configuration.vars.customer %}
+
+<div class="col-12 col-md">
+    <div class="d-md-flex gap-2 align-items-center">
+        {{ header.h1(customer.firstName ~ ' ' ~ customer.lastName) }}
```

Comment:
```suggestion
        {{ header.h1('sylius.ui.orders'|trans) }}
```

Since this is the collection page, I’d suggest displaying the simple `Orders` label. Using first and last names could result in an empty title, as these fields are not required. The email, however, is always available and can reliably be used instead.